src/main/java/Calculator.java

```diff
@@ -1,27 +1,32 @@
+
 public class Calculator {
     public static int add(String a, String b) {
         double i = stringToDouble(a);
         double j = stringToDouble(b);
-        return (int) (i + j);
+        if(i + j > Integer.MAX_VALUE) throw new IllegalArgumentException("[ERROR]결과값이 Integer 범위 이상입니다.");
```

Comment:
이렇게 줄 바꿈이 되는 것 보다는 
```suggestion
if (i + j > Integer.MAX_VALUE) {
    throw new IllegalArgumentException("[ERROR]결과값이 Integer 범위 이상입니다.");
}
```
이런 식의 코드가 가독성이 좋을 것 같은데 어떻게 생각하시나요?

Reply:
그리고 현재 모든 연산에서 정수 상한선(Integer.MAX_VALUE)의 대한 고려는 되어있는데 하한선(Integer.MIN_VALUE)에 대한 고려는 되어있지 않은 것 같습니다! (음수에 대한 고려가 없는 것 같습니다!)

Reply:
현재 인풋은 소수점을 받아도 처리가 가능하도록 되어있는데 반환 값이 int면 올바른 결과 값이 나올 수 없는 계산기인 것 같습니다.
예를 들면 1.1, 1.2가 인풋으로 들어와 덧셈이 이루어진다면 사용자는 2.3을 의도할 것 같은데 현재 계산 결과는 int형으로 반환되어 올바른 결과가 나타나지 않을 것 같습니다. 
(소수 입력은 고려하지 않아도 될 것 같습니다)

Reply:
> 그리고 현재 모든 연산에서 정수 상한선(Integer.MAX_VALUE)의 대한 고려는 되어있는데 하한선(Integer.MIN_VALUE)에 대한 고려는 되어있지 않은 것 같습니다! (음수에 대한 고려가 없는 것 같습니다!)

헉 엄청난 실수를 했네요...반영하도록 하겠습니다!

Reply:
> 이렇게 줄 바꿈이 되는 것 보다는
> 
> 이런 식의 코드가 가독성이 좋을 것 같은데 어떻게 생각하시나요?

여러 사람과 공유하는 코드는 길이를 줄이는 것보다 제안해주신 방식이 더 보기 편할것 같아요ㅎㅎ